autogpt_platform/backend/backend/integrations/oauth/base.py

```diff
@@ -7,6 +7,7 @@
 
 class BaseOAuthHandler(ABC):
     PROVIDER_NAME: ClassVar[str]
+    DEFAULT_SCOPES: ClassVar[list[str]] = []
```

Comment:
No reason for this to be part of the base interface, pls just keep it on the Google one

Reply:
Discussed, leaving here